code/modules/reagents/reagent_dispenser.dm

```diff
@@ -525,6 +525,37 @@
 	flags = FPRINT | TWOHANDABLE | MUSTTWOHAND // If I end up being coherent enough to make it holdable in-hand
 	var/list/exiting = list() // Manages people leaving the barrel
 	health = 50
+	var/burning = FALSE
+	var/list/comfyfire = list('sound/misc/comfyfire1.ogg','sound/misc/comfyfire2.ogg','sound/misc/comfyfire3.ogg',)
+	var/list/possible_fuels = list(
```

Comment:
instead of pasting, make this a global list and call it in both places; more maintainable

Reply:
Implemented in latest commit

Reply:
you did it with the comfy fire sounds but not the much larger fuel list

Reply:
ah, didn't realize you were referring to both. will fix in next commit